src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable {
+    protected Color color;
+    protected double area;
+
+    public Figure(Color color, double area) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. However, in this case, since 'area' should be calculated within each specific figure class, you might not need to pass it to the 'Figure' constructor at all.